apps/galaxy/src/modules/Orders/Details.tsx

```diff
@@ -0,0 +1,129 @@
+                .quotationDetails.map(
+                  (quotationDetail: QuotationDetails) =>
+                    `[${quotationDetail.color}, ${quotationDetail.quantity}] `
+                )}
+            </Text>
+            <br />
+            <Text fontWeight="bold">Design Preview</Text>
+            <Image src={info.getValue().constructionDraw} alt="design preview alt" />
+            <br />
+            <Text fontWeight="bold">Court Details</Text>
+            <TableContainer>
+              <Table variant="simple">
+                <Tbody>
+                  <Tr>
+                    <Td paddingLeft="0px">design name</Td>
+                    <Td>{info.getValue().design.designName}</Td>
+                  </Tr>
+                  <Tr>
+                    <Td paddingLeft="0px">tile color</Td>
+                    <Td>
+                      {info
+                        .getValue()
+                        .design.tileColor.map(
+                          (tile: TileColor) => `[${tile.color}, ${tile.location}] `
+                        )}
+                    </Td>
+                  </Tr>
+                  <Tr>
+                    <Td paddingLeft="0px">length</Td>
+                    <Td>{info.getValue().design.courtSize.length.toString()}</Td>
+                  </Tr>
+                  <Tr>
+                    <Td paddingLeft="0px">width</Td>
+                    <Td>{info.getValue().design.courtSize.width.toString()}</Td>
+                  </Tr>
+                  <Tr>
+                    <Td paddingLeft="0px">three point radius</Td>
+                    <Td>{info.getValue().design.courtSize.threePointRadius.toString()}</Td>
+                  </Tr>
+                  <Tr>
+                    <Td paddingLeft="0px">three point line</Td>
+                    <Td>{info.getValue().design.courtSize.threePointLine.toString()}</Td>
+                  </Tr>
+                  <Tr>
+                    <Td paddingLeft="0px">length of corner</Td>
+                    <Td>{info.getValue().design.courtSize.lengthOfCorner.toString()}</Td>
+                  </Tr>
+                  <Tr>
+                    <Td paddingLeft="0px">restricted area length</Td>
+                    <Td>{info.getValue().design.courtSize.restrictedAreaLength.toString()}</Td>
+                  </Tr>
+                  <Tr>
+                    <Td paddingLeft="0px">side border width</Td>
+                    <Td>{info.getValue().design.courtSize.sideBorderWidth.toString()}</Td>
+                  </Tr>
+                  <Tr>
+                    <Td paddingLeft="0px">line border width</Td>
+                    <Td>{info.getValue().design.courtSize.lineBorderWidth.toString()}</Td>
+                  </Tr>
+                </Tbody>
+              </Table>
+            </TableContainer>
+            <br />
+            <Text fontWeight="bold">Need To Level Ground</Text>
+            <Text>{info.getValue().isNeedLevelGround.toString()}</Text>
```

Comment:
I guess best not to display just `true || false`, `Yes || No` would make more sense to the end user